ios/FluentUI/Core/Colors.swift

```diff
@@ -257,6 +257,35 @@ public final class Colors: NSObject {
             }
         }
 
+        public var colorSet: ColorSet {
+            switch self {
+            // Set of colors defined as global tokens
+            case .communicationBlue:
+                return GlobalTokens.Colors.Brand.primary.value
+            case .communicationBlueTint40:
+                return GlobalTokens.Colors.Brand.tint40.value
+            case .communicationBlueTint30:
+                return GlobalTokens.Colors.Brand.tint30.value
+            case .communicationBlueTint20:
+                return GlobalTokens.Colors.Brand.tint20.value
+            case .communicationBlueTint10:
+                return GlobalTokens.Colors.Brand.tint10.value
+            case .communicationBlueShade30:
+                return GlobalTokens.Colors.Brand.shade30.value
+            case .communicationBlueShade20:
+                return GlobalTokens.Colors.Brand.shade20.value
+            case .communicationBlueShade10:
```

Comment:
This is a prototype for an effort to move away from asset catalogs for Fluent colors. These colors are defined in the global token set, and it's cheaper to fetch those directly than go through the asset catalog. However, I didn't remove them from the catalog yet because I haven't scrubbed through all the places that might reference them by name. Eventually all vNext colors should be in this switch, and the asset catalog can either be removed, or can just contain legacy colors.